scripts/linux-x86_64/install.sh

```
@@ -2,16 +2,89 @@
+      if [ -z "$2" ] || [ "${2#-}" != "$2" ]; then
+        echo "Error: --install-dir requires a non-empty directory path"
+        exit 1
+      fi
+      if printf '%s' "$2" | grep -q '[[:space:]]'; then
+        echo "Error: --install-dir must not contain whitespace"
+        exit 1
+      fi
+      INSTALL_DIR="$2"
+      shift 2
+      ;;
+      ;;
```

Shellcheck (0.11.0)

[error] Expected 'esac' to close the case statement. Fix any mentioned problems and try again.

(SC1072)